pcsx2/Docs/PlayStation_2_Memory_Card_File_System.md

```diff
@@ -0,0 +1,965 @@
+# PlayStation 2 Memory Card File System
+
+<div style="clear: both">
+
+## By [Ross Ridge](https://plus.google.com/101505960668383360394/?rel=author)
+
+## Public Domain
+
+</div>
+
+<div style="clear: both">
+
+This document is a description of the file system layout as used on PlayStation 2 memory cards. It's based on the research I did while writing [mymc](http://www.csclub.uwaterloo.ca/%7Erridge/mymc), a utility for working with PS2 memory card images. This document tries to be comprehensive an accurate, but some details may be missing, misleading or just plain wrong. At lot of assumptions had to be made during my research, and it's hard know to what exactly Sony intended in every case. All most all of the names for structures, fields and flags were made up by me. Nothing in this document should be considered official.
+
+For brevity, unused fields and flag bits are omitted from the tables. In most cases unused fields or flags should be assumed to be either reserved or padding and set to zero when writing. In particular, you'll need to pad out the structures to the length given at the top of the table. All values are stored on the card using the little-endian byte order.
+
+</div>
+
+<div style="clear: both">
+
+## NAND Flash Memory Basics
+
+<div style="float: right; padding: 1em 0 1em 1em; width: 40%">
+
+### Glossary
+
+A number of terms are used in this document that you may not familiar with or are different from other sources describing PS2 memory cards, so I've created short glossary.
+
+<dl>
```

Comment:
Completely unreadable when viewing the markdown as markdown.
I suggest using a list like this example to create the definitions table: https://meta.stackexchange.com/a/298872